### PR TITLE
fix bug #5456(Cannot display feed back icon and prefix icon together)

### LINKED ISSUE #5456: Cannot display Form.Item feed icon(hasFeedback) when Input has prefix
#### Environment(required)

- antd version: 2.8.2
- OS and its version: ubuntu 16.04 LTS
- Browser and its version: Chrome Version 56.0.2924.87 (64-bit)

#### What did you do? Please provide steps to re-produce your problem.
Use Form.Item hasFeedback and Input prefix together.
```js
<FormItem
      {...formItemLayout}
      label="Fail"
      hasFeedback
      validateStatus="error"
      help="Should be combination of numbers & alphabets"
    >
      <Input
        prefix={<Icon type="user" style={{ fontSize: 13 }} />}
        placeholder="input prefix icon with feed icon" id="error" />
 </FormItem>
```

#### What do you expected?
The feedback icon can be displayed.

#### What happen?
Cannot display feedback icon.

### PR DESCRIPTION
**Pull request by william-yz**
fix [#5456](https://github.com/ant-design/ant-design/issues/5456)

### CODE REVIEW

File: components/form/demo/validate-static.md [hunk 1]
```diff
@@ -94,6 +94,19 @@ ReactDOM.render(
...
+      validateStatus="error"
+      help="Should be combination of numbers & alphabets"
+    >
+      <Input
+        prefix={<Icon type="user" style={{ fontSize: 13 }} />}
+        placeholder="input prefix icon with feed icon" id="error"
+      />
+    </FormItem>
+
+    <FormItem
```
Comment (afc163) [inline components/form/demo/validate-static.md]: CI is broken, and no need to change demo.

File: components/form/style/index.less [hunk 2]
```diff
@@ -337,6 +337,7 @@ form {
     font-size: @font-size-lg;
     animation: zoomIn .3s @ease-out-back;
     .iconfont-font("");
+    z-index: 2;
```
Comment (afc163) [inline components/form/style/index.less]: Maybe `z-index: 1` is enough to fix it?

**Comment by afc163** [comment]
It would be nice to make your PR title readable.

**Comment by benjycui** [comment]
Why do Codecov need to worry about styles' code..

**Pull request merged**